Automattic/jetpack
# Block Upgrade banner: Updated the upgrade banner description to include the plan name

Added the plan name in the message of the upgrade banner so that users would be informed of what plan they need to purchase to use a specific block.





Fixes [#46141](https://github.com/Automattic/wp-calypso/issues/46141)

#### Changes proposed in this Pull Request:

* Includes the plan name in the upgrade banner message

#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






* Apply D73873-code on your sandbox
* Go to a free Simple site and add a premium block
* Notice that the upgrade banner is "Upgrade to <plan> to use this premium block"

**Before:**
![Screenshot 2022-01-27 at 12 40 10](https://user-images.githubusercontent.com/995955/151343249-dcd28613-5b1a-4e6c-9e85-8b2bed0cf9a8.png)

**After:**
![Screenshot 2022-01-27 at 12 39 09](https://user-images.githubusercontent.com/995955/151343336-4fec32bd-43f5-4a83-855e-3aff10b9117c.png)
